src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/ScoreVariantAnnotations.java

```diff
@@ -0,0 +1,322 @@
+                    : null;
+            final File indelScorerPklFile = new File(modelPrefix + ".indel.scorer.pkl");
+            final File indelNegativeScorerPklFile = new File(modelPrefix + ".indel.negative.scorer.pkl");
+            indelScorer = indelScorerPklFile.canRead()
+                    ? indelNegativeScorerPklFile.canRead()
+                        ? VariantAnnotationsScorer.combinePositiveAndNegativeScorer(
+                                new PythonSklearnVariantAnnotationsScorer(pythonScriptFile, indelScorerPklFile),
+                                new PythonSklearnVariantAnnotationsScorer(pythonScriptFile, indelNegativeScorerPklFile))
+                        : new PythonSklearnVariantAnnotationsScorer(pythonScriptFile, indelScorerPklFile)
+                    : null;
+        } else {
+            logger.info("Python script was not provided, running in BGMM mode...");
+            // TODO validate BGMM model inputs
+
+            // TODO extract method and constants
+            final File snpScorerSerFile = new File(modelPrefix + ".snp" + BGMMVariantAnnotationsModel.BGMM_SCORER_SER_SUFFIX);
+            final File snpNegativeScorerSerFile = new File(modelPrefix + ".snp.negative" + BGMMVariantAnnotationsModel.BGMM_SCORER_SER_SUFFIX);
+            snpScorer = snpScorerSerFile.canRead()
+                    ? snpNegativeScorerSerFile.canRead()
+                        ? VariantAnnotationsScorer.combinePositiveAndNegativeScorer(
+                                BGMMVariantAnnotationsScorer.deserialize(snpScorerSerFile),
+                                BGMMVariantAnnotationsScorer.deserialize(snpNegativeScorerSerFile))
+                        : BGMMVariantAnnotationsScorer.deserialize(snpScorerSerFile)
+                    : null;
+            final File indelScorerSerFile = new File(modelPrefix + ".indel" + BGMMVariantAnnotationsModel.BGMM_SCORER_SER_SUFFIX);
+            final File indelNegativeScorerSerFile = new File(modelPrefix + ".indel.negative" + BGMMVariantAnnotationsModel.BGMM_SCORER_SER_SUFFIX);
+            indelScorer = indelScorerSerFile.canRead()
+                    ? indelNegativeScorerSerFile.canRead()
+                        ? VariantAnnotationsScorer.combinePositiveAndNegativeScorer(
+                                BGMMVariantAnnotationsScorer.deserialize(indelScorerSerFile),
+                                BGMMVariantAnnotationsScorer.deserialize(indelNegativeScorerSerFile))
+                        : BGMMVariantAnnotationsScorer.deserialize(indelScorerSerFile)
+                    : null;
+        }
+
+        // TODO validate modes
+        if (snpScorer == null && indelScorer == null) {
+            throw new UserException.BadInput("At least one serialized scorer must be present in model files.");
+        }
+
+        // TODO extract method and constants
+        final File snpTruthScores = new File(modelPrefix + ".snp.truthScores.hdf5");
```

Comment:
@meganshand here's where you can read in the truth scores for each variant type and do whatever calculations you need. Check the VariantAnnotationsScorer interface for the `static double[] readScores(final File inputFile)` method, which you can use to read in the scores.